modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -0,0 +1,430 @@
+
+. Make sure there is a dedicated Aura user that will be used for fetching metrics
+ ** Either create a new user:
+  ... In https://console.neo4j.io/#user-management["User Management"^] of Neo4j Aura, invite a new user, selecting "Metrics Integration Reader" as a role.
+image:invite_user.png[]
+  ... Follow the invitation link and log in to Neo4j Aura.
+  ... Confirm the tenant membership.
+ ** Alternatively, find an existing user in https://console.neo4j.io/#user-management["User Management"^] and change its role to "Metrics Integration Reader" +
++
+____
+*_NOTE_:* Capabilities of users having the role "Metrics Integration Reader" are limited to fetching the metrics and getting a read-only view of the tenant.
+____
+. In https://console.neo4j.io/#account["Account Details"^], create new Aura API credentials. Save client secret. +
+image:api_credentials.png[]
+. Configure the APM system to fetch metrics from the URL(s) or configuration templates shown in https://console.neo4j.io/#metrics-integration["Metrics Integration"^] of Neo4j Aura. Use `oauth2` type of authentication specifying the Client ID and Client Secret created in the previous step. See examples for <<example-of-using-prometheus,Prometheus and Grafana>> and <<example-of-using-datadog,Datadog>> below.
+. Use APM system to create visualizations, dashboards and alarms based on Neo4j metrics.
+
+[#security]
+== Security
+
+Metrics for a Neo4j Aura instance will only be returned if all the following is true:
+
+* `Authorization` header of the metrics request contains a valid token,
+* token was issued for Aura user having "Metrics Integration Reader" role,
+* Tenant is of type `enterprise`,
+* specified instance belongs to the specified tenant.
+
+[#revoke-access-to-metrics]
+=== Revoke access to metrics
+
+To revoke user's access to metrics of a specific tenant, remove the user from that tenant in https://console.neo4j.io/#user-management["User Management"^]. After that, the user will still exist but its connection to the tenant will be removed.
+
+____
+*_NOTE_:* The revocation described above takes effect after the authorization caches expire, which takes approximately 5 minutes.
+It results in HTTP 401 being returned, along with the message `User doesn't have access to Metrics resources.`
+____
+
+____
+*_NOTE_:* If, instead of removing the user, only the Aura API credentials used to retrieve metrics are removed, the revocation will take effect only after the tokens issued with these credentials expire, as no new token can be issued anymore. Currently used token expiration time is 1 hour.
+____
+
+[#metric-labels]
+=== Metric labels
+
+Depending on the metric, following labels are applied:
+
+* `aggregation`: the aggregation used to calculate the metric value, set on every metric. Since the Neo4j instance is deployed as a Neo4j cluster, aggregations are performed to combine values from all relevant cluster nodes. Currently used aggregations: `MIN`, `MAX`, `AVG` and `SUM`.
```

Comment:
```suggestion
* `aggregation`: the aggregation used to calculate the metric value, set on every metric. 
Since the Neo4j instance is deployed as a Neo4j cluster, aggregations are performed to combine values from all relevant cluster nodes. 
The following aggregations are used: `MIN`, `MAX`, `AVG` and `SUM`.
```